eclass/systemd.eclass

```diff
@@ -227,6 +227,53 @@ systemd_install_serviced() {
 	)
 }
 
+# @FUNCTION: systemd_install_override
+# @USAGE: [--user] <conf-file> [<service>]
+# @DESCRIPTION:
+# Install <conf-file> as the override file <service>.d/00gentoo.conf.
+# If <service> is not specified
+# <conf-file> with the .conf suffix stripped is used
+# (e.g. foo.service.conf -> foo.service.d/00gentoo.conf).
+# Defaults to systemd unit overrides, unless --user is provided.
+# The required argument <conf-file> may be '-', in which case the
+# file is read from stdin and <service> must also be specified.
+# @EXAMPLE:
+# systemd_install_override "${FILESDIR}/foo.service.conf"
+# systemd_install_override - foo.service <<-EOF
+# 	[Service]
+# 	RestartSec=120
+# EOF
+systemd_install_override() {
```

Comment:
systemd docs use the term "drop-in", so I would prefer to use that instead for "override".